docs/syntax/imports.md

```diff
@@ -146,3 +163,63 @@ exports that create name clashes must be resolved at the _export_ site.
 
 Bindings exported from a module `X` are available in an identical fashion to
 bindings that are _defined_ in the module `X`.
+
+### Implicit exports
+
+The compiler inserts implicit exports for entities defined in a module and for
+submodules of a _synthetic module_. A synthetic module is basically a directory
+in the source structure.
+
+#### Defined entities in a module
+
+Entities defined in a module are automatically exported from the module. This
+means that the following modules are semantically identical:
+
+```
+type My_Type
+method x = x
+```
+
+```
+export project.Module.My_Type
+export project.Module.method
```

Comment:
Should we add a _"duplicit export"_ error/warning later?

Reply:
Why not. I have created issue for that in https://github.com/enso-org/enso/issues/10514